core/commands/add.go

```diff
@@ -29,6 +31,7 @@ const progressReaderIncrement = 1024 * 256
 const (
 	progressOptionName = "progress"
 	wrapOptionName     = "wrap-with-directory"
+	hiddenOptionName   = "hidden"
```

Comment:
i'm terrible at UX, but i expect dotfiles and hidden files to be ignored by default